processor/configuration/src/main/resources/schemas/sources/catalog.json

```diff
@@ -22,5 +22,11 @@
     "isConnector": true,
     "name": "Slack Source",
     "description": "Ingest data from a Slack channel."
+  },
+  {
+    "id": "azure_eventhubs_source_0.1",
+    "name": "Azure EventHub Source",
```

Comment:
Reading other strings, I am not sure about `EventHub` vs `EventHubs`?

Reply:
same as before.